frontend/src/views/FinishedView.vue

```diff
@@ -29,6 +36,18 @@ const props = defineProps({
 const sequence = ref(
   await SequenceRestInterface.getMetadataForStudent(props.code)
 );
+
+/**
+ * Opens a new tab and downloads certificate as pdf
+ */
+function downloadCertificate(): void {
+  window.open(
+    BaseRestInterface.getBaseURL() +
+      '/sequence/' +
```

Comment:
This URL should be gotten from the sequence rest interface since we use the /sequence/ here